### PR TITLE
[Ready] Syndie air alarm/APC subtypes, cleaning up syndie lavaland piping

### PR DESCRIPTION
**Pull request by 81Denton**
:cl: Denton
code: Added /syndicate subtypes for air alarms and APCs.
fix: Added missing stock parts to the syndicate lavaland base.
tweak: Added kitchen related circuit boards to the syndie lavaland base.
/:cl:

1) Added /syndicate subtypes for air alarms + APCs and replaced existing ones where possible.
2) Replaced layer 3 piping in the syndie lavaland base with /layer3 subtypes.
3) Added missing stock parts to the base, included some kitchen related circuit boards and upgraded the chem dispensers, so players can use said stock parts for building deep fryers instead of upgrading the dispensers.

### CODE REVIEW

File: code/modules/atmospherics/machinery/airalarm.dm [hunk 1]
```diff
@@ -153,6 +153,10 @@
 	req_access = null
 	req_one_access = null
 
+/obj/machinery/airalarm/syndicate //general syndicate access
+	req_access = null
+	req_access_txt = "150"
```
Comment (ShizCalev) [inline code/modules/atmospherics/machinery/airalarm.dm]: just `req_access = list(ACCESS_SYNDICATE)`

File: code/modules/power/apc.dm [hunk 2]
```diff
@@ -98,6 +98,10 @@
 
 /obj/machinery/power/apc/unlocked
 	locked = FALSE
+	
+/obj/machinery/power/apc/syndicate //general syndicate access
+	req_access = null
+	req_access_txt = "150"
```
Comment (ShizCalev) [inline code/modules/power/apc.dm]: same here.

**Review by ShizCalev — CHANGES_REQUESTED** [review]

**Comment by SpaceManiac** [comment]
Are you feeling up to changing the engine? Fusion EMPs means ignition of the turbine after it's been sitting for any length of time will blow the entire base.

**Comment by 81Denton** [comment]
The incinerator would probably have to be replaced with RTGs in another PR. Boring gameplay wise, but I can't think of any engine that's compact and won't blow up the entire base.

**Comment by 81Denton** [comment]
Actually, I couldn't get the syndie turbine into fusion range at all. Maybe the temperature is too low in general?

**Comment by 81Denton** [comment]
I've run some tests and while the chamber gets up ~31.500 K, it never hits a power_ratio where EMPs and explosions start occuring.

**Comment by 81Denton** [comment]
Anywho, the incinerator creating EMPs large enough to blow the base were a bug that's been fixed a while ago.

File: code/game/objects/items/storage/boxes.dm [hunk 3]
```diff
@@ -966,3 +966,47 @@ obj/item/storage/box/clown
 /obj/item/storage/box/holy_grenades/PopulateContents()
 	for(var/i in 1 to 7)
 		new/obj/item/grenade/chem_grenade/holy(src)
+
+/obj/item/storage/box/stockparts //for ruins where it's a bad idea to give access to an autolathe/protolathe, but still want to make stock parts accessible
+	name = "box of stock parts"
+	desc = "Contains a variety of basic stock parts."
+
+/obj/item/storage/box/stockparts/basic
```
Comment (SpaceManiac) [inline code/game/objects/items/storage/boxes.dm]: Any reason this isn't just part of `box/stockparts`? Not sure it needs to change, just wondering if you had a reason.

Comment (81Denton) [reply code/game/objects/items/storage/boxes.dm]: I did the parent/subtype thing out of habit, no real reason behind it. Want me to change it?

Comment (SpaceManiac) [reply code/game/objects/items/storage/boxes.dm]: Might as well. I get the feeling that having fewer redundant typepaths floating about the codebase is a good idea.

**Pull request merged**